log/internal/rollbar/rollbar.go

```diff
@@ -26,13 +26,19 @@ func Setup(l *zap.SugaredLogger, service string) func() {
 
 	env := os.Getenv("PACKET_ENV")
 	if env == "" {
-		log.Panicw("required envvar is unset", "envvar", "PACKET_ENV")
+		env = os.Getenv("PKG_ENV")
+		if env == "" {
+			log.Panicw("required envvar is unset", "envvar", "PKG_ENV")
+		}
 	}
 	rollbar.SetEnvironment(env)
 
 	v := os.Getenv("PACKET_VERSION")
 	if v == "" {
-		log.Panicw("required envvar is unset", "envvar", "PACKET_VERSION")
+		v = os.Getenv("PKG_VERSION")
```

Comment:
ditto, `env.Get("PKG_VERSION", env.Get("PACKET_VERSION"))`

Reply:
I've been wanting to push this down from `log.Init` and have the calling binary figure out how to get the version info. It doesn't really make sense for the binary's version # to come from the env.